db/seeds.rb

```diff
@@ -79,15 +79,54 @@ def position(type)
   surname = player['web_name']
   full_name = "#{player['first_name']} #{player['second_name']}"
   position = position(player['element_type'])
+
+   # Get specific player url
+   url = 'https://fantasy.premierleague.com/drf/element-summary/' << player['id'].to_s
```

Comment:
use string interpolation here, it's more consistent with the rest of our code and is much easier if we need to put a variable in the middle of a string

Reply:
is that + rather than <<?